react/features/device-selection/DeviceSelectionPopup.js

```diff
@@ -61,9 +61,9 @@ export default class DeviceSelectionPopup {
             disableAudioInputChange: true,
             disableDeviceChange: true,
             hasAudioPermission: JitsiMeetJS.mediaDevices
-                .isDevicePermissionGranted('audio'),
+                .isDevicePermissionGranted.bind('audio'),
             hasVideoPermission: JitsiMeetJS.mediaDevices
-                .isDevicePermissionGranted('video'),
+                .isDevicePermissionGranted.bind('video'),
```

Comment:
I think here maybe you meant to bind _this_ to null (or not the string you passed in) and bind the argument to the string. So .bind(null, audio) maybe?